test/layers.jl

```diff
@@ -175,7 +171,7 @@ video = Video(600, 600)
     rm("layer_test.gif")
 
 
-    Javis.CURRENT_VIDEO[1] = vid
```

Comment:
I think this was to reset the CURRENT_VIDEO but that wasn't a clean way. This is better now.